packages/env/lib/build-docker-compose-config.js

```diff
@@ -51,6 +51,10 @@ module.exports = function buildDockerComposeConfig( config ) {
 		...themeMounts,
 	];
 
+	// Set the default ports based on the config values.
+	const mainPort = `\${WP_ENV_PORT:-${ config.portNumber }}:80`;
+	const testsPort = `\${WP_ENV_TESTS_PORT:-${ config.testsPortNumber }}:80`;
```

Comment:
Let's name this `testsPorts` (add the `s`), because we're passing it into `ports` below.

Reply:
Done :)